src/libYARP_logger/include/yarp/logger/YarpLogger.h

```diff
@@ -174,7 +174,14 @@ class yarp::yarpLogger::LogEntry
     bool                          append_logEntry(MessageEntry entry);
 
     public:
-    LogEntry                       (int _entry_list_max_size=10000) {logging_enabled=true;entry_list_max_size=_entry_list_max_size; last_read_message=-1;entry_list.reserve(entry_list_max_size);};
+    LogEntry                       (int _entry_list_max_size=10000) :
+        logging_enabled(true),
+        entry_list_max_size(_entry_list_max_size),
+        last_read_message(-1),
+        entry_list_max_size_enabled(true)
+    {
+        entry_list.reserve(entry_list_max_size);
+    }
```

Comment:
Sorry, I didn't mean that you had to change this with my previous comment...
Anyway it looks way better now. 👍
Since you are changing this, can you please remove the space between `LogEntry` and `(` as well?

Reply:
I am sure I will have to merge it properly on https://github.com/robotology/yarp/pull/1396... 😢

Reply:
@drdanz ok I've done this but you have to know that the rest of the file is written with previous style :laughing: 
I also rebased the branch over the master.